R/ggplot_as_list.R

```diff
@@ -55,8 +58,9 @@ ggplot.as.list <- function(o, ...) {
     }
     p <- list()
     for (i in 1:l) {
-      t <- paste("canvas", i, sep = "-")
-      p[[i]] <- gg_cxplot(o$plots[[i]]$fn(d, o$plots[[i]]$mapping), t)
+      t      <- paste("canvas", i, sep = "-")
+      p[[i]] <- gg_cxplot(o$plots[[i]], t)
+
       p[[i]]$isGGMatrix <- cx$longestString
```

Comment:
@cb4ds - should not this be set to TRUE?

Reply:
I don't know, sorry, I don't have any context here to answer that.